Dockerfile

```diff
@@ -13,6 +17,12 @@ RUN bundle install --jobs 4 && npm install && npm install -g phantomjs-prebuilt
 
 EXPOSE 3000
 ADD . /myapp
-RUN RAILS_ENV=production bundle exec rake assets:precompile
+RUN if [ -z "$CUSTOM_ASSETS_URL" ]; then\
```

Comment:
Hey @Tuuleh, I looked into docker docs, and I believe the ENV instruction doesn't really work for us. ENV allows to define vars *in* the Dockerfile, but it doesn't support setting them from outside the docker file. I'm not being able to find a way to pass configuration options to the `build` task in any other way but --build-arg.